cudax/examples/CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ endfunction()
 file(GLOB example_srcs
   RELATIVE "${cudax_SOURCE_DIR}/examples"
   CONFIGURE_DEPENDS
-  *.cu
```

Comment:
Is that still needed?

Reply:
not for this pr, no. i can keep that change local.